CHANGES.rst

```diff
@@ -58,6 +58,8 @@ Imviz
 - Catalog Search: When catalog is imported from file, its original column names are
   preserved on export. [#3519]
 
+- User API for Catalog Search plugin is now public. [#3529]
```

Comment:
Please list the method names because users don't usually code dive a PR. Thanks!